app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: float,
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car_list: list) -> int:
+        price = 0
+        for car in car_list:
+            if car.clean_mark < self.clean_power:
+                price += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return price
+
+    def calculate_washing_price(self, car: Car) -> float:
+        c_p = self.clean_power
+        a_r = self.average_rating
+        c_m = car.clean_mark
+        distant = self.distance_from_city_center
+        car_comfort = car.comfort_class
```

Comment:
no need to make a new variables for already exsiting variables, also u are using bad names for variables
Try to split ur formula to a few parts, for better readability